wordpress-mobile/WordPress-iOS
# Reader post comments: don't show Comments table if there are no comments.

Ref: #17511 
Depends on: #17603

The Comments table is not shown unless there are post comments.

To test:
- Go to Reader > post with no comments > details.
- Verify the Comments table is not displayed.
- Go to Reader > post with comments > details.
- Verify the Comments table is displayed.

| No Comments | Comments |
|--------|-------|
| ![none](https://user-images.githubusercontent.com/1816888/144151470-4bdb8d73-3a3f-4dcc-be41-0931a43433cb.png) | ![comments](https://user-images.githubusercontent.com/1816888/144151463-729c936e-935a-49b2-bce1-4ad6e4068c20.png) |

## Regression Notes
1. Potential unintended areas of impact
N/A. WIP.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. WIP.

3. What automated tests I added (or what prevented me from doing so)
N/A. WIP.

PR submission checklist:
